crates/gwt-core/runtime/chroma_index_runner.py

```
@@ -2830,6 +2836,384 @@ def action_index_board_v2(
+        try:
+            old_entries = read_manifest(db_path, scope="works")
+            diff = compute_manifest_diff(old_entries, new_entries)
+            source_changed = bool(diff["added"] or diff["changed"] or diff["removed"])
+
+            if mode != "incremental" or source_changed:
+                try:
+                    existing = collection.get()
+                    if existing.get("ids"):
+                        collection.delete(ids=existing["ids"])
+                except Exception:
+                    pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)